src/main/java/com/faforever/client/chat/ChannelTabController.java

```diff
@@ -579,12 +581,13 @@ public void addSearchFieldListener() {
   }
 
   public void onAdvancedUserFilter(ActionEvent actionEvent) {
+    advancedUserFilter.setSelected(userFilterController.isFilterAppliedProperty().getValue());
```

Comment:
just use property.get() instead of getValue() as above, getValue() returns a Double, get() a double, here the primitive double is expected, won't change any behaviour but would add more consistency.

Reply:
Or just introduce a primtive boolean getter.